### PR TITLE
Separate the ELB subnet at existing VPC

### PR DESCRIPTION
**Pull request by kubotak-is**
Hi @gwdp 

I will a request to separate the ELB subnet in the existing VPC.
For example, Fargate is a private subnet and ELB is a public subnet.

### CODE REVIEW

**Comment by gwdp** [comment]
Hi @kubotak-is, thanks for the contribution. I'll be happy to merge it and publish a new version. Just one question first :) 

I recall trying to place my tasks on a private subnet but ECS could not fetch the image and neither the metadata services/ Have you tested it already? Does your VPC have any special endpoints to make it work?

Also, I have been migrating this plugin to be more generic for ECS usage and have included a bunch of new features at https://github.com/Hybridless/serverless-ecs-plugin; Still not stable and neither fully documented, but new release should be out next week :) So, stay tuned there as well.

**Comment by kubotak-is** [comment]
@gwdp Thanks for the reply.

In the case of a private subnet, you can set up nat-gateway so that Fargate can get the image.
I have confirmed this by actually running it.

You may not need the nat-gateway by using PrivateLink, but I have not verified that.

File: src/options.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ export interface IVPCOptions {
     vpcId: string;
     securityGroupIds: string[];
     subnetIds: string[];
+    elbSubnetIds: string[];
```
Comment (gwdp) [inline src/options.ts]: Sorry for taking to long. About to merge, could you fix this, please? I believe this needs to be optional. `elbSubnetIds?: string[];`

**Review by gwdp — CHANGES_REQUESTED** [review]

**Comment by gwdp** [comment]
> @gwdp Thanks for the reply.
> 
> In the case of a private subnet, you can set up nat-gateway so that Fargate can get the image.
> I have confirmed this by actually running it.
> 
> You may not need the nat-gateway by using PrivateLink, but I have not verified that.

Perfect, thanks for letting me know, will check this and probably add to the documentation.

File: src/options.ts (hunk 1 above)
Comment (kubotak-is) [reply src/options.ts]: thanks. fixed it

**Comment by gwdp** [comment]
@kubotak-is `1.9.4` is available at npm now. Thanks for contributing.
Don't forget, this plugin is getting unsupported in the future and you should rely on the repository I mentioned earlier.
It already has these changes and alb VPC support :)

**Pull request merged**